btree/btree.c

```diff
@@ -2530,13 +2532,8 @@ static bool sibling_node_check(struct m0_btree_oimpl *oi)
 	return true;
 }
 
-static bool locked(struct td *tree)
-{
-	return false;
-}
-
 static int64_t lock_op_init(struct m0_sm_op *bo_op, struct node_op  *i_nop,
-			    struct td       *tree, int nxt)
+			    struct td *tree, int nxt)
 {
 	/** parameters which has passed but not will be used while state machine
```

Comment:
Is it necessary to return state while calling lock_op_init()?
As I see through the code, we are anyways falling through to the next state in every occasion.

Reply:
In future if we implement separate  state machine for locks the return statement will be needed as we might get nxt as WAIT.